README.md

```diff
@@ -225,10 +225,14 @@ Emitted when a peer is removed.
 
 Emitted when the feed is appended to, either locally or remotely.
 
-#### `feed.on('download', seq)`
+#### `feed.on('download', seq, byteLength)`
```

Comment:
byteLength here means we break with normal hypercore which contains the buffer. i don't personally mind this, as this is prob a more flexible api, just fyi @andrewosh

Reply:
Cool. Think it's fine considering it's an instance of the same API difference (not pushing buffers to the client), which could already require some special handling.

Reply:
Accttualllyyy chatted about it a bit, and it's probably best for `byteLength` to be the 3rd arg, with a null `data` argument in between, just to avoid the unnecessary breakage. 

`feed.on('download', seq, data, byteLength)` where data is `null`, and perhaps an option to make it non-null in the future

Reply:
Done